deploy-service.sh

```diff
@@ -2,7 +2,7 @@
 set -eu
 
 rm -rf govuk-cdn-config-secrets
-git clone git@github.com:alphagov/govuk-cdn-config-secrets.git
+git clone git@github.com:alphagov/govuk-cdn-config-secrets.git -b add_www_staging_values
```

Comment:
This needs to point to master before merging.

Reply:
yes, thanks, we know. It is in the trello card before we merge etc